server/src/main/java/module-info.java

```diff
@@ -4,7 +4,8 @@
     requires com.google.protobuf;
     requires io.grpc.stub;
     requires io.helidon.common;
+    requires io.helidon.config;
     requires io.helidon.webserver.grpc;
     requires io.helidon.webserver;
-    requires java.logging;
+    requires jakarta.inject;
```

Comment:
Interesting we have Jakarta Inject in the module info, but basically don't use it.
Did we intend to use CDI and it's just not in place yet?

Reply:
It's currently being used [here](https://github.com/hashgraph/hedera-block-node/blob/def04528948c7a75c9e42e0d7c2fa36ab479ba05/server/src/main/java/com/hedera/block/server/persistence/cache/LRUCache.java#L29).  I anticipate we will make more use of it in the future.